modules/simul/src/main/JsonView.scala

```diff
@@ -86,7 +86,7 @@ final class JsonView(
       },
       "name"       -> simul.name,
       "fullName"   -> simul.fullName,
-      "variants"   -> simul.variants.map(variantJson(chess.Speed(simul.clock.config.some))),
+      "variants"   -> simul.variants.map(variantJson(chess.Speed(simul.clock.participantConfig.some))),
```

Comment:
Whenever there is a single config to be chosen, we use the participants config (since simuls are always unrated,
this should not be too important, and this is consistent with the old behavior imho